src/main/java/com/nha/abdm/wrapper/common/dataPackaging/keys/KeyController.java

```diff
@@ -0,0 +1,72 @@
+/* (C) 2024 */
+package com.nha.abdm.wrapper.common.dataPackaging.keys;
+
+import com.nha.abdm.wrapper.common.dataPackaging.Constants;
+import java.security.*;
+import org.bouncycastle.asn1.x9.X9ECParameters;
+import org.bouncycastle.crypto.ec.CustomNamedCurves;
+import org.bouncycastle.jce.interfaces.ECPrivateKey;
+import org.bouncycastle.jce.interfaces.ECPublicKey;
+import org.bouncycastle.jce.provider.BouncyCastleProvider;
+import org.bouncycastle.jce.spec.ECParameterSpec;
+import org.springframework.stereotype.Service;
+
+@Service
+public class KeyController {
+  private String senderPublicKey;
+  private String senderPrivateKey;
+  private String senderNonce;
+
+  public KeyMaterial fetchKeys() throws Exception {
+    if ((senderPrivateKey == null && senderPublicKey == null) && senderNonce == null) {
```

Comment:
if all are &&, then we dont need extra bracket. this can be `if (senderPrivateKey == null && senderPublicKey == null && senderNonce == null) {`